src/Core/Managed/Shared/Extensibility/Implementation/Metrics/MetricTerms.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics
+{
+    using System;
+
+    internal static class MetricTerms
+    {
+        private const string MetricPropertiesNamePrefix = "Microsoft.ApplicationInsights.Metrics";
+
+        public static class Aggregation
+        {
+            public static class Interval
+            {
+                public static class Moniker
```

Comment:
I personally find it confusing when reading code that looks like a namespace and then isn't because it's harder to locate where things come from when you don't have a full IDE with intellisense running (and I run into that situation regularly).  I admit that it is a nitpick and personal preference but in this case I have a strong preference for constants with long names.